app/javascript/topNavigation/utilities.js

```diff
@@ -1,6 +1,7 @@
 function closeHeaderMenu(memberMenu, menuNavButton) {
   menuNavButton.setAttribute('aria-expanded', 'false');
-  memberMenu.classList.remove('desktop', 'showing', 'clicked');
+  memberMenu.classList.remove('desktop', 'showing');
+  memberMenu.removeAttribute('data-clicked');
```

Comment:
Refactor (non-blocking): delete the property from the dataset.

```suggestion
  delete memberMenu.dataset.clicked;
```